docs/src/tablespaces.md

```diff
@@ -125,12 +125,12 @@ status:
 
 ## Storage classes and tablespaces
 
-As for PGDATA and WAL volumes, you can use different storage classes for your
-tablespaces too. This is a very convenient way of optimizing your resources,
+As you can for PGDATA and WAL volumes, you can use different storage classes for your
+tablespaces. This is a convenient way of optimizing your resources,
```

Comment:
Maybe this would be simpler to parse?
```suggestion
You can use different storage classes for your tablespaces, just as you can for PGDATA and
WAL volumes. This is a convenient way of optimizing your resources,
```